packaging/homebrew/shopify-cli.base.rb

```diff
@@ -13,7 +13,7 @@
 class ShopifyCli < Formula
   module RubyBin
     def ruby_bin
-      Formula["ruby"].opt_bin
+      Formula["ruby@3.0"].opt_bin
```

Comment:
I probably didn't see this on my system due to having Ruby 3.1 installed already. Thanks for fixing.